specs/app.spec.js

```diff
@@ -0,0 +1,149 @@
+const fs = require('fs'),
+    mysql = require('mysql'),
+    parse = require('csv-parse'),
+    expect = require('chai').expect,
+    DB_CONFIG = require('../dbconfig.json'),
+    sinon = require('sinon');
+
+describe('When the user has csv files in the files folder', function () {
+    this.timeout(10000);
+
+    beforeEach(function () {
+        return ensureTablesRemoved(['t1', 't2', 'tr']);
```

Comment:
`tr` means table in the root, I have created this file to ensure that when I use the flag --files and the program runs it will insert this file instead of inserting the files in the root directory